tsconfig.json

```diff
@@ -0,0 +1,23 @@
+
+{
+  "compilerOptions": {
+    "outDir": "./built",
```

Comment:
outDir should be `dist` or `build`

Reply:
Done, changed to `dist`. 
I had followed [TS Handbook Migrating from JavaScript
](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html) for the name previously.